### PR TITLE
ICP-7022 Re-enable emergency heat as a supported mode

### PR DESCRIPTION
**Pull request by greens**

### CODE REVIEW

**Comment by greens** [comment]
verified by SR-Pol.

**Review by tpmanley — DISMISSED** [review]

File: devicetypes/smartthings/zwave-battery-thermostat.src/zwave-battery-thermostat.groovy [hunk 1]
```diff
@@ -303,7 +303,7 @@ def zwaveEvent(physicalgraph.zwave.commands.thermostatmodev2.ThermostatModeSuppo
 	if(cmd.heat) { supportedModes << "heat" }
 	if(cmd.cool) { supportedModes << "cool" }
 	if(cmd.auto) { supportedModes << "auto" }
-//	if(cmd.auxiliaryemergencyHeat) { supportedModes << "emergency heat" } // device doesn't actually support this
+	if(cmd.auxiliaryemergencyHeat) { supportedModes << "emergency heat" } // device doesn't actually support this
```
Comment (dkirker) [inline devicetypes/smartthings/zwave-battery-thermostat.src/zwave-battery-thermostat.groovy]: So is it just that the device flat out doesn't support it, or doesn't support us setting it to this?

**Review by dkirker — DISMISSED** [review]

File: devicetypes/smartthings/zwave-battery-thermostat.src/zwave-battery-thermostat.groovy (hunk 1 above)
Comment (greens) [reply devicetypes/smartthings/zwave-battery-thermostat.src/zwave-battery-thermostat.groovy]: I suppose I should remove that comment.

**Comment by greens** [comment]
Comment removed. Please re-approve.

**Review by dkirker — APPROVED** [review]

**Pull request merged**